### PR TITLE
Fix some issues related "config"

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix some issues related to config

### CODE REVIEW

File: gateway/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+cryptography>=42.0.5
```
Comment (Tansito) [inline gateway/requirements.txt:1]: For what do we need this dependency, Aki?

Comment (akihikokuroda) [reply gateway/requirements.txt:1]: I have no idea why this started happening.

Comment (Tansito) [reply gateway/requirements.txt:1]: Oh, I saw the error in PR that I just opened. I will take a look 👀

File: gateway/tests/api/test_v1_program.py [hunk 2]
```diff
@@ -106,14 +106,6 @@ def test_run(self):
                 "arguments": arguments,
                 "dependencies": "[]",
                 "env_vars": env_vars,
-                "config": json.dumps(
```
Comment (Tansito) [inline gateway/tests/api/test_v1_program.py:109]: I want to propose modify the test not remove. It should be something as simple as send the `json`, should not?

File: gateway/requirements.txt (hunk 1 above)
Comment (IceKhan13) [reply gateway/requirements.txt:1]: yep, I also noticed

File: gateway/tests/api/test_v1_program.py (hunk 2 above)
Comment (akihikokuroda) [reply gateway/tests/api/test_v1_program.py:109]: This test should put a `dict` here but the test client `APITestCase.client.post` doesn't support **multipart uploads with nested data**.  So I took it out for now.

Comment (Tansito) [reply gateway/tests/api/test_v1_program.py:109]: Sorry @akihikokuroda , I didn't have the time to see your comment until now. I tested it locally and for me is working the `json.dumps`, is it returning you an error?

Comment (akihikokuroda) [reply gateway/tests/api/test_v1_program.py:109]: Based on https://github.com/Qiskit-Extensions/quantum-serverless/pull/1297#discussion_r1589287485.  I take put this "test_run" test :-)

Comment (Tansito) [reply gateway/tests/api/test_v1_program.py:109]: Hahaha fair point. Let's do that then 👍

**Review by Tansito — APPROVED** [review]
LGTM 👍

**Pull request merged**